Autopilot/PathManager/Src/pathStateClasses.cpp

```diff
@@ -120,4 +324,360 @@ pathManagerState& fatalFailureMode::getInstance()
     return singleton;
 }
 
+/****************************************************************************************************
+LANDING STATE FUNCTIONS
+****************************************************************************************************/
+
+void landingTransitionStage::execute(pathManager* pathMgr)
+{
+    //load in sensor fusion data and telemtry data into input structure
+    input.telemetryData = *(commsWithTelemetry::GetTelemetryIncomingData());
```

Comment:
Why are we passing these structs by value ? They're read only. They should be passed as constant references or something C++y of the sort.